snooty.toml

```diff
@@ -4,5 +4,4 @@ toc_landing_pages = ["/fundamentals/connection", "/fundamentals/crud"]
 
 [constants]
 version = 4.0
-package-name-org = "mongodb-org"
```

Comment:
Not sure what this change is for, but I'll take your word for it.

Reply:
Thanks. I think this got in here while I was working on something else.